zeroconf/__init__.py

```diff
@@ -2339,7 +2345,7 @@ def unregister_all_services(self) -> None:
                 i += 1
                 next_time += _UNREGISTER_TIME
 
-    def check_service(self, info: ServiceInfo, allow_name_change: bool) -> None:
+    def check_service(self, info: ServiceInfo, allow_name_change: bool, cooperating_responders: bool) -> None:
```

Comment:
This is essentially a breaking change, why not default to `False`?

Reply:
Only if `check_service()` is a public API I think? (I'm not entirely sure if it should be or not)

Reply:
I assume that everything is public unless it starts with an underscore :) But yes, I don't remember whether this function is intended to be public.